jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/JpaWebServer.java

```diff
@@ -10,7 +10,7 @@
 @Table(name = "webserver", uniqueConstraints = {@UniqueConstraint(columnNames = {"name"})})
 @NamedQueries({
         @NamedQuery(name = JpaWebServer.FIND_WEB_SERVER_BY_QUERY,
-                query = "SELECT ws FROM JpaWebServer ws WHERE ws.name = :wsName"),
+                query = "SELECT ws FROM JpaWebServer ws WHERE lower(ws.name) = :wsName"),
```

Comment:
Can we do this? lower(ws.name) = lower(:wsName), so we don't have to do .toLowerCase() in Java. May me also change the name to FIND_WEB_SERVER_BY_NAME